File: app/src/main/java/com/BlizzardArmory/diablo/item/Secondary.java
package com.BlizzardArmory.diablo.item;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class Secondary {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("textHtml")
    @Expose
    private String textHtml;

    public String getText() {
        return text;
    }

    public String getTextHtml() {
        return textHtml;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/diablo3/item/Secondary.kt
package com.BlizzardArmory.model.diablo.diablo3.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Secondary.
 */
@Keep
data class Secondary(

    @SerializedName("text")

    val text: String,

    @SerializedName("textHtml")

    val textHtml: String

)